test/built-ins/Temporal/PlainDateTime/prototype/withPlainDate/non-compatible-calendars-throw.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.withplaindate
+description: If two non-ISO calendars are involved, an error is raised
+features: [Temporal]
+---*/
+
+const dt = new Temporal.PlainDateTime(1995, 12, 7, 3, 24, 30);
+
+const cal = {
+  id: 'foo',
+  toString() { return "this is a string"; },
+};
+
+const anotherCal = {
+  id: 'bar',
+  toString() { return "this is another string"; },
+};
+
+const date = new Temporal.PlainDate(2008, 9, 6, anotherCal);
+const dtWithNewCalendar = dt.withCalendar(cal);
```

Comment:
You could dispense with the `withCalendar` call and just create `dt` here with `cal` already in the constructor arguments.

Reply:
done